enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md

```diff
@@ -52,18 +54,22 @@ cd caf-terraform-landingzones-starter/enterprise_scale/construction_sets/aks
 configuration_folder=online/aks_secure_baseline/configuration
 
 # Define the configuration files to apply, all tfvars files within the above folder recursively
```

Comment:
Define the configuration files to apply, all tfvars files within the above folder recursively except for launchpad subfolder which is not relevant for this standalone guide.